src/TLuaInterpreter.cpp

```diff
@@ -15835,3 +15836,106 @@ int TLuaInterpreter::getMouseEvents(lua_State * L)
     }
     return 1;
 }
+
+int TLuaInterpreter::setMudletConfig(lua_State * L)
```

Comment:
The more concise `setPreference` sounds nice to me - as well as the ability to parse a table for keys, so you can use one call to set everything. It's a neater API design. Need help with parsing the table from C++ side?